adaptive_breakpoints/test/adaptive_breakpoints_test.dart

```diff
@@ -0,0 +1,84 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+
+
+import 'package:flutter/foundation.dart';
```

Comment:
Move these imports to be with the others, delete all the extra new lines

Reply:
Done